README.md

```diff
@@ -383,10 +527,24 @@ documentation](#documenation) before you ask for help. Especially the
 <a name="release-notes"></a>
 ## Release notes
 
-Please see the [GitHub
-releases page](https://github.com/Dynatrace/OneAgent-SDK-for-Python/releases),
-and the [PyPI release
-history](https://pypi.org/project/oneagent-sdk/#history).
+Please see the [GitHub releases page](https://github.com/Dynatrace/OneAgent-SDK-for-Python/releases),
+and the [PyPI release history](https://pypi.org/project/oneagent-sdk/#history).
+
+### Breaking Changes in 1.1.0
```

Comment:
call it "Upgrading to 1.1"?

Reply:
Since we are still in EAP, I would remove this section totally and add this (useful!) information in the release notes comment.

From Beta&GA, we should have it documented on this page, but for EAP release notes as enough.

Reply:
fixed

Reply:
it's already documented in the drafted release notes for 1.1.0, too